src/Faker/Provider/fa_IR/Person.php

```diff
@@ -0,0 +1,116 @@
+        'لادن', 'لاله', 'لبینا', 'لطیفه', 'لُعبت', 'لعیا', 'لوما', 'لیان', 'لیدا', 'لیلا', 'لیلا', 'لیلاس', 'لیلوپر', 'لیلی', 'لیلی', 'لیلیا', 'لیما', 'لینا',
+        'مائده', 'مارال', 'ماریه', 'مامک', 'مامیسا', 'مانا', 'ماندا', 'ماندیس', 'مانلی', 'مانی', 'مانیا', 'ماه آفرد', 'ماهدخت', 'ماهرخ', 'ماهرخسارر', 'ماهرو', 'محبوبه', 'محیا', 'مدینه', 'مراجل', 'مرجان', 'مرجانه', 'مرسده', 'مرضیه', 'مرمر', 'مروارید', 'مریم', 'مژده', 'مژگان', 'مستانه', 'مستوره', 'مشیا', 'معصومه', 'ملاحت', 'ملکه', 'ملکه جهان', 'ملود', 'ملودی', 'ملوس', 'ملوک', 'ملیح', 'ملیحه', 'ملیسا', 'ملیکا', 'منصوره', 'منظر', 'منوّر', 'منیر', 'منیره', 'منیژه', 'منیلا', 'مه سیما', 'مه منیر', 'مها', 'مهان', 'مهبانو', 'مهتا', 'مهتاب', 'مهتاج', 'مهجبین', 'مهداد', 'مهدخت', 'مهدیس', 'مهدیه', 'مهرآرا', 'مهرآسا', 'مهرآفرین', 'مهرا', 'مهراز', 'مهران', 'مهراندخت', 'مهرانگیز', 'مهرانه', 'مهراور', 'مهراوه', 'مهربانو', 'مهرخ', 'مهردخت', 'مهرزاد', 'مهرسا', 'مهرشید', 'مهرک', 'مهرناز', 'مهرنوش', 'مهرورز', 'مهروش', 'مهری', 'مهزاد', 'مهسا', 'مهسان', 'مهستی', 'مهسو', 'مهشاد', 'مهشید', 'مهفام', 'مهکامه', 'مهلا', 'مهلقا', 'مهناز', 'مهنوش', 'مهوش', 'مهین', 'مهین بانو', 'مهیندخت', 'موجان', 'موژان', 'مونا', 'مونا', 'میترا', 'میثاق', 'میثمه', 'میچکا', 'میشا', 'میلا', 'میلی', 'مینا', 'مینو', 'میهن', 'میهن یار',
+        'ناجی', 'نادره', 'نادی', 'نادیا', 'نارسیس', 'نارین', 'ناز', 'نازآفرین', 'نازبانو', 'نازبو', 'نازپری', 'نازتا', 'نازدانه', 'نازدخت', 'نازک', 'نازگل', 'نازلی', 'نازنوش', 'نازنین', 'نازو', 'نازی', 'نازیتا', 'نازیلا', 'نازینه', 'ناژو', 'ناژین', 'ناناز', 'ناهید', 'ناهیده', 'نجلا', 'نجمه', 'نجوی', 'ندا', 'نرجس', 'نرگس', 'نرمین', 'نرمینه', 'نزهت', 'نسا', 'نسترن', 'نسرین', 'نسیم', 'نشاط', 'نشوا', 'نشید', 'نظیره', 'نغمه', 'نفیسه', 'نکیسا', 'نگار', 'نگارین', 'نگان', 'نگاه', 'نگین', 'نهال', 'نهاله', 'نوا', 'نواز', 'نوال', 'نوبر', 'نور', 'نورا', 'نورانگیز', 'نوژان', 'نوشآفرین', 'نوشا', 'نوشابه', 'نوشبر', 'نوشه', 'نوشین', 'نوشینه', 'نوگل', 'نویده', 'نویسه', 'نوین', 'نیاز', 'نیاز', 'نیّر', 'نیرا', 'نیّره', 'نیسا', 'نیسیا', 'نیکا', 'نیکپر', 'نیکتا', 'نیکدخت', 'نیکدل', 'نیکناز', 'نیکی', 'نیکی', 'نیکی ناز', 'نیلگون', 'نیلوفر', 'نینا', 'نیوشا', 'نیوشه',
+        'هاله', 'هانا', 'هاني', 'هانيه', 'هایده', 'هدیه', 'هستی', 'هلاله', 'هما', 'همای', 'همراز', 'هنگامه', 'هوردخت', 'هورشید', 'هیلا', 'هیلا', 'هیلدا', 'هیوا',
+        'وارسته', 'واژه', 'واله', 'وانوشه', 'وجستا', 'وجیهه', 'ورتا', 'ورد', 'وردا', 'ورسا', 'ورنا', 'وستا', 'وشتا', 'وشتی', 'وصال', 'وندا', 'ونیژه', 'ویدا', 'ویرا', 'ویره', 'ویژه', 'ویس', 'ویستا', 'ویشکا', 'وینا',
+        'یارا', 'یاس', 'یاسمن', 'یاسمین', 'یاقوت', 'یزدانه', 'یکامه', 'یکتا', 'یگانه', 'یلدا', 'یوتاب',
+    );
+
+    protected static $lastName = array(
```

Comment:
I didn't found any specific source for last names, just added the names that i know.